tests/collapse.spec.ts

```diff
@@ -0,0 +1,72 @@
+import { test, expect } from '@playwright/test';
+
+test('Collapse E2E Tests', async ({ page }) => {
+  await page.goto('/test.html');
+
+  // All collapse elements should be visible
+  const collapse = page.getByTestId('collapse-default')
```

Comment:
We should probably add a test case for a collapse with breadcrumbs and a badge like this
`<v-collapse
  title='Motor 1'
  variant=''
>
  <v-breadcrumbs slot='title' crumbs='Robot,Motor'></v-breadcrumbs>
  <v-badge slot='header' label='Inactive'></v-badge>
  <div style='font-size: 12px; padding: 1rem'>Motor one was concieved and executed at Bell Labs in 1972 under the guidance of lead director Dennis Richie and Superviser Wallace Breen.</div>
</v-collapse>`

Reply:
to make sure all the extra elements render correctly

Reply:
added!